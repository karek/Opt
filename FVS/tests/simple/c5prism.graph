a1 a2
a2 a3
a3 a4
a4 a5
a5 a1
b1 b2
b2 b3
b3 b4
b4 b5
b5 b1
a1 b1
a2 b2
a3 b3
a4 b4
a5 b5

