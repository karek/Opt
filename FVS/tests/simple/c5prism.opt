3
a1 a3 b4 
