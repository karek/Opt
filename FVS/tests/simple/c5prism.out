3
a5 b2 b4 
